CRM/Contact/Form/Contact.php

```diff
@@ -818,8 +818,7 @@ public function buildQuickForm() {
     CRM_Contact_Form_Location::buildQuickForm($this);
 
     // add attachment
-    $this->addElement('file', 'image_URL', ts('Browse/Upload Image'), 'size=30 maxlength=60');
-    $this->addUploadElement('image_URL');
+    $this->addField('image_URL', array('type' => 'File', 'size' => '30', 'maxlength' => '60', 'label' => ts('Browse/Upload Image')));
```

Comment:
`'type' => 'File'` can probably be defined in the schema, as well as size and maxlength.